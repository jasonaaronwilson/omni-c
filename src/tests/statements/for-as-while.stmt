for(;1;){break;}