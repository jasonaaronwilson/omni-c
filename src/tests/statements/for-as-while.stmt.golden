{
    tag = PARSE_NODE_FOR_STATEMENT
    for_init = {
        tag = PARSE_NODE_EMPTY_STATEMENT
    }
    for_test = {
        tag = PARSE_NODE_LITERAL
        tokens = [
            1
        ]
    }
    for_body = {
        tag = PARSE_NODE_BLOCK
        statements = [
            {
                tag = PARSE_NODE_BREAK_STATEMENT
                break_keyword_token = break
            }
        ]
    }
}

// C Output
for (
  ;
  1;
  )
{
  break;
}

